### DISCUSSION TITLE
How to have dashes in page names?

### DISCUSSION THREAD

**Discussion by JakubKoralewski** [vercel/next.js#15396] [Help] [answered]
I have a page like `hello_there.tsx`, but I would like it to be accessible under the URL `/hello-there`. Is there a way to do this?

**Comment by awareness481** [comment, marked as answer]
Does Typescript have a limitation for dashes in .tsx files? Because .js filenames with dashes work just fine.
ex. hello-test.js will create a /hello-test URL

**Reply by JakubKoralewski** [reply]
~~Yes~~, either my setup prohibits dashes or TypeScript itself. When I try to import a filename with dashes I get: `TS2307: Cannot find module 'hello-there' or its corresponding type declarations.`.

Nevermind, I used the wrong path 😅. It's supported.

**Reply by awareness481** [reply]
Oh alright, it happens :P
